tox.ini

```diff
@@ -2,6 +2,8 @@
 # in multiple virtualenvs. This configuration file will run the
 # test suite on all supported python versions. To use it, "pip install tox"
 # and then run "tox" from this directory.
+# Users running macOS may need to install libomp.  If using the Homebrew 
+# package manager then this can be achieved with "brew install libomp".
```

Comment:
```suggestion
# Users running macOS may need to install libomp.  If using the Homebrew
# package manager then this can be achieved with "brew install libomp".
```

Reply:
The unittest failed because of the trailing whitespace.